src/jit/codegencommon.cpp

```diff
@@ -4426,32 +4447,32 @@ void            CodeGen::genFnPrologCalleeRegArgs(regNumber xtraReg,
 
         if (doingFloat)
         {
-#if defined(_TARGET_ARM_) || defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
-            insCopy = ins_Copy(TYP_FLOAT);
+#if defined(FEATURE_HFA) || defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
+            insCopy = ins_Copy(TYP_DOUBLE);
             // Compute xtraReg here when we have a float argument
```

Comment:
The argument to Ins_Copy is only used to select the type of instruction fmov vs. mov
The size of the argument float vs. double makes no difference here. 
I prefer to use TYP_DOUBLE everywhere rather than TYP_FLOAT.